docs/getting-started/new_project_guide.md

```diff
@@ -368,9 +370,8 @@ information on code coverage generation.
 
 
 **Note:** Currently, we only support AddressSanitizer (address) and UndefinedBehaviorSanitizer (undefined)
-configurations. MemorySanitizer is recommended, but needs to be enabled manually once you verify
-that all system dependencies are
-[instrumented](https://github.com/google/oss-fuzz/blob/master/infra/base-images/msan-libs-builder/Dockerfile#L20).
+configurations by default.
+MemorySanitizer is recommended, but needs to be enabled manually since you must build all libraries with MemorySanitizer.
```

Comment:
s/libraries/runtime dependencies/ to be more clear (and on line 125).

Reply:
Done.